spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
+ActiveRecord::Migration.maintain_test_schema!
```

Comment:
Well, it's no big deal. We can leave it as it is, a rm it in another commit, for reference.

According to the docs, since Rails 4.1 it's no longer necessary: https://github.com/rails/rails/blob/ce392d948403a4b5cb7348669cb597a3c93b8d3c/guides/source/upgrading_ruby_on_rails.md#changes-to-test-helper